.github/actions/changes/get_pr_info.sh

```diff
@@ -105,6 +105,7 @@ if [[ "${GITHUB_EVENT_NAME}" == "push" ]] && [[ "$BORS" == false || ( "$BRANCH"
   if [[ -n "$QUERIED_GITHASH" ]] && [[ $(git merge-base --is-ancestor "$QUERIED_GITHASH" "$(git rev-parse HEAD)" 2>/dev/null; echo $?) == 0 ]]; then
     BASE_GITHASH="${QUERIED_GITHASH}"
   fi
+  TARGET_BRANCH=${BRANCH}
```

Comment:
But it should be set here, if it was a push to a non bors branch, or if bors isn't in the picture.